dashboard/src/components/jobs/parameters/TimeParameters.vue

```diff
@@ -42,11 +69,19 @@ export default class JobTimeParameters extends Vue {
   timezoneList: Array<string> = Timezones;
 
   data() {
+    const zone = new LocalZone().name;
+    // @ts-expect-error
+    const now = DateTime.now()
+      .setZone(zone)
+      .set({
+        second: 0,
+        milliseconds: 0
+      });
     return {
-      start: "2020-01-01T00:00+00:00",
-      end: "2020-02-01T00:00+00:00",
+      start: now.toISO(),
+      end: now.toISO(),
```

Comment:
Do we need to set these to not null initially? One issue is that the start picker max time is restricted to end, so in a normal workflow you wouldn't be able to pick a start in the future (and end after start). No reason we should restrict the range if not necessary

Reply:
I've updated these to initialize to null and the "Get Started" button is disabled until the user makes a selection for start and end.